mbtools/remove_styles.py

```diff
@@ -0,0 +1,40 @@
+from pathlib import Path
+from . import models
+from lxml.etree import tostring
+import argparse
+
+
+def remove_styles(mbz_dir):
+    mbz_path = Path(mbz_dir).resolve(strict=True)
+    moodle_backup = models.MoodleBackup(mbz_path)
+    activities = moodle_backup.activities()
+    for activity in activities:
+        for elem in activity.html_elements():
+            elem.remove_attr("style")
+        with open(activity.activity_filename, "w") as f:
+            f.write(tostring(activity.etree,
+                             pretty_print=True,
+                             encoding='utf8').decode())
```

Comment:
> 3\. Can you remind me what the difference in output is when passing `pretty_print=True` versus not?

pretty_print enables formatted XML. It adds whitespaces between nodes from what I understand.

Reply:
> 4\. I think it might be good to have a single utility / shared function that writes etrees

Good idea, I have updated PR 91 with a function in utils to do this.